dev/devicelab/README.md

```diff
@@ -226,3 +227,33 @@ target for each operating system.
 
 Flutter's DeviceLab has a limited capacity in presubmit. File an infra ticket
 to investigate feasibility of adding a test to presubmit.
+
+## Migrating to build and test model
+
+To better utilize limited DeviceLab testbed resources and speed up overal validation
+time, it is now supported to separate building artifacts (.apk/.app) from testing them.
+The artifact will be built on a host only bot no needing a device, and the test will run
```

Comment:
```suggestion
The artifact will be built on a host only bot, a VM without a device, and the test will run
```

Reply:
Updated with `, a VM or physical bot without`. For Linux it is a vm, but for Mac it is a physical bot.